### ISSUE TITLE
Why sierra template speech style is lucas arts?

### ISSUE THREAD

**Issue by ericoporto** [adventuregamestudio/ags-template-source#36]
https://github.com/adventuregamestudio/ags-template-source/blob/b44355ee7212b20f02fdd495b041add13188dd31/Sierra-style/Game.agf#L75

Is there a particular reason for this? Has it always been like this?

Edit:

https://github.com/adventuregamestudio/ags-template-source/blob/2981329e54d29003ed9e3b3ccbd561664f3f28c3/Default%20Game/Game.agf#L47

Turns out it's been like this since the first commit. That's a bit unintuitive, but I guess it's one the things that are kept for historical purposes.

**Comment by morganwillcock** [comment]
Maybe it should be changed.

It probably wasn't considered when the template was renamed, but now the description doesn't match what it does. I think that would be OK if there is a second template that demonstrates the other style.

**Comment by ericoporto** [comment]
I just checked and VerbCoin, BASS and Tumbleweed also use LucasArts speech style.

**Comment by ivan-mogilko** [comment]
Of course, because that's the simpliest possible style where you do not have to provide any other graphical assets; so it works well as default.

For sierra-speech you need to have at least proper speech views, otherwise it won't work and display la speech again.

**Comment by morganwillcock** [comment]
Maybe someone would be willing to make the additional speech views so that the same "asset pack" could potentially work across every template (and eventually in a modular future, allow easy switching between the styles)?

**Comment by ivan-mogilko** [comment]
> Maybe someone would be willing to make the additional speech views so that the same "asset pack" could potentially work across every template (and eventually in a modular future, allow easy switching between the styles)?

Agh, we should've posted this request on forums, nobody sees this here.
I will do this now.

**Comment by ivan-mogilko** [comment]
But there's a problem to keep in mind: you cannot have these portraits assigned to a SpeechView and have Lucas-arts speech, because then they will be displayed as a character animation.

How do you do a switch between the styles gracefully then? Looks like this would require some setup in game_start, like a selection between two speech views. This will additionally complicate a template's script.
